cmd/template/nri-resmgr-template-deployment.yaml.in

```diff
@@ -0,0 +1,142 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nri-resmgr
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nri-resmgr
+rules:
+- apiGroups:
+  - ""
+  - nriresmgr.intel.com
+  resources:
+  - nodes
+  - configmaps
+  - labels
+  - annotations
+  verbs:
+  - get
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - topology.node.k8s.io
+  resources:
+  - noderesourcetopologies
+  verbs:
+  - create
+  - get
+  - list
+  - update
+  - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: nri-resmgr
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: nri-resmgr
+subjects:
+- kind: ServiceAccount
+  name: nri-resmgr
+  namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: nri-resmgr
+  name: nri-resmgr
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: nri-resmgr
+  template:
+    metadata:
+      labels:
+        app: nri-resmgr
+    spec:
+      serviceAccount: nri-resmgr
+      nodeSelector:
+        kubernetes.io/os: "linux"
+      containers:
+        - name: nri-resmgr-template
+          args:
+            - --host-root
+            - /host
+            - --fallback-config
+            - /etc/nri-resmgr/nri-resmgr.cfg
```

Comment:
`nri-resmgr` directory is changed to `nri-resource-policy` and filename to `nri-resource-policy.cfg`